high_card.rb

```diff
@@ -3,3 +3,62 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+=begin
+1. Build a deck of shuffled cards.
+	* The deck should be an array of cards.
+	* Each card should be a mini-array with a value and suit. Example: `[8, "hearts"]`
+	* Shuffle the deck. It would look something like this...
+	```ruby
+	deck = [
+		[8, "hearts"],
+		[2, "diamonds"],
+		["J", "clubs"],
+		...
+	]
+	```
+
+deck = []
+
+ranks.each do |rank|
+  suits.each do |suit|
+    card = "#{rank} #{suit.to_s}"
+    ary = Array.new
+    Array.new(card)
+    #deck.each_with_index do |rank, suit|
+    #end
+  end
```

Comment:
this looks good, I think better than string might be to use a complex data type like array or hash.